src/vs/workbench/contrib/interactiveSession/browser/interactiveSessionContributionServiceImpl.ts

```diff
@@ -15,7 +15,7 @@ import { SyncDescriptor } from 'vs/platform/instantiation/common/descriptors';
 import { Registry } from 'vs/platform/registry/common/platform';
 import { ViewPaneContainer } from 'vs/workbench/browser/parts/views/viewPaneContainer';
 import { IViewContainersRegistry, IViewDescriptor, IViewsRegistry, ViewContainer, ViewContainerLocation, Extensions as ViewExtensions } from 'vs/workbench/common/views';
-import { getClearAction, getOpenInteractiveSessionEditorAction } from 'vs/workbench/contrib/interactiveSession/browser/actions/interactiveSessionActions';
+wimport { getClearAction } from 'vs/workbench/contrib/interactiveSession/browser/actions/interactiveSessionActions';
```

Comment:
It looks to me like the change from "import" to "wimport" is a mistake, I suggest removing the w.